scripts/convert_hf_checkpoint.py

```diff
@@ -179,36 +179,28 @@ def copy_weights_phi(
     dtype: Optional[torch.dtype] = None,
 ) -> None:
     weight_map = {
```

Comment:
Can you include an error where we check if the user has the old weights and instruct to redownload the data?

Reply:
I can do this.